shared/api/network.go

```diff
@@ -58,3 +58,23 @@ type NetworkLease struct {
 	Address  string `json:"address" yaml:"address"`
 	Type     string `json:"type" yaml:"type"`
 }
+
+// NetworkStateStats represents statistics about an interface
+//
+// API extension: network_state
+type NetworkStateStats struct {
+	BytesTransmitted   int64 `json:"bytes_transmitted" yaml:"bytes_transmitted"`
+	BytesReceived      int64 `json:"bytes_received" yaml:"bytes_received"`
+	PacketsTransmitted int64 `json:"packets_transmitted" yaml:"packets_transmitted"`
+	PacketsReceived    int64 `json:"packets_received" yaml:"packets_received"`
+}
+
+// NetworkState represents the state of an interface
+//
+// API extension: network_state
+type NetworkState struct {
+	MTU       int               `json:"mtu" yaml:"mtu"`
```

Comment:
This should be called `Mtu` for consistency (an unfortunate one though) with `ContainerStateNetwork`